src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Filters/SegmentFiltersTests.cs

```diff
@@ -29,7 +29,21 @@ public class SegmentFiltersTests
 ORC|RE||35508^NIST-AA-IZ-2|||||||7824^Jackson^Lily^Suzanne^^^^^NIST-PI-1^L^^^PRN|||||||NISTEHRFAC^NISTEHRFacility^HL70362
 RXA|0|1|20131112||88^influenza, unspecified formulation^CVX|999|||01^Historical Administration^NIP001|||||||||||CP|A";
 
-        private static readonly Hl7v2Data TestData = LoadTestData();
+        private const string TestDataContent2 = @"MSH|^~\&|AccMgr|1|||20050110045504||ADT^A01|599102|P|2.3||| 
```

Comment:
Can we use `TestDataContent` for the test? If not, consider renaming the new data instead of adding index after it.